src/test/java/com/fasterxml/jackson/databind/BaseMapTest.java

```diff
@@ -268,7 +268,7 @@ protected void assertStandardEquals(Object o)
         throws IOException
     {
         String str = m.writeValueAsString(value);
-        return (Map<String,Object>) m.readValue(str, Map.class);
+        return (Map<String,Object>) m.readValue(str, LinkedHashMap.class);
```

Comment:
Not really needed; default implementation is `LinkedHashMap`. Although maybe it's ok being more explicit.